src/app/@ansyn/ol/plugins/annotations/annotations-context-menu/annotations-context-menu.module.ts

```diff
@@ -6,15 +6,17 @@ import { AnnotationsWeightComponent } from './components/annotations-weight/anno
 import { AnnotationsColorComponent } from './components/annotations-color/annotations-color.component';
 import { ColorPickerComponent } from './components/color-picker/color-picker.component';
 import { ColorPickerModule } from 'ngx-color-picker';
-import { MatInputModule } from '@angular/material';
+import { MatInputModule, MatSelectModule } from '@angular/material';
```

Comment:
MatSelectModule  - > check if we have something working in our app with our style

Reply:
can't use combo-box from @ansyn/ansyn inside @ansyn/ol